### ISSUE TITLE
Class `craft\commerce\Plugin` not found

### ISSUE THREAD

**Issue by chrismlusk** [nystudio107/craft-seomatic#980]
Updated to v3.4.9 and now get the following error when attempting to edit an entry in the CP:

```
Error: Class 'craft\commerce\Plugin' not found in /Users/chrislusk/Code/ODC/oak-street-health/vendor/nystudio107/craft-seomatic/src/services/MetaContainers.php:891
Stack trace:
#0 /Users/chrislusk/Code/ODC/oak-street-health/vendor/nystudio107/craft-seomatic/src/services/MetaContainers.php(207): nystudio107\seomatic\services\MetaContainers->getAllowedUrlParams()
#1 /Users/chrislusk/Code/ODC/oak-street-health/vendor/nystudio107/craft-seomatic/src/services/MetaContainers.php(395): nystudio107\seomatic\services\MetaContainers->loadMetaContainers('become-a-patien...', 1)
#2 /Users/chrislusk/Code/ODC/oak-street-health/vendor/nystudio107/craft-seomatic/src/fields/SeoSettings.php(366): nystudio107\seomatic\services\MetaContainers->previewMetaContainers('become-a-patien...', 1, true)
#3 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/fieldlayoutelements/CustomField.php(244): nystudio107\seomatic\fields\SeoSettings->getInputHtml(Object(nystudio107\seomatic\models\MetaBundle), Object(craft\elements\Entry))
#4 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/fieldlayoutelements/BaseField.php(203): craft\fieldlayoutelements\CustomField->inputHtml(Object(craft\elements\Entry), false)
#5 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/fieldlayoutelements/CustomField.php(213): craft\fieldlayoutelements\BaseField->formHtml(Object(craft\elements\Entry), false)
#6 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/web/View.php(1499): craft\fieldlayoutelements\CustomField->craft\fieldlayoutelements\{closure}()
#7 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/fieldlayoutelements/CustomField.php(214): craft\web\View->namespaceInputs(Object(Closure), 'fields')
#8 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/models/FieldLayout.php(573): craft\fieldlayoutelements\CustomField->formHtml(Object(craft\elements\Entry), false)
#9 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/web/View.php(1494): craft\models\FieldLayout->craft\models\{closure}()
#10 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/models/FieldLayout.php(572): craft\web\View->namespaceInputs(Object(Closure), NULL)
#11 /Users/chrislusk/Code/ODC/oak-street-health/vendor/twig/twig/src/Extension/CoreExtension.php(1511): craft\models\FieldLayout->createForm(Object(craft\elements\Entry), false, Array)
#12 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/helpers/Template.php(106): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\models\FieldLayout), 'createForm', Array, 'method', false, false)
#13 /Users/chrislusk/Code/ODC/oak-street-health/storage/runtime/compiled_templates/0d/0dd09a2132f50fa91ecd174f8855920be2f00a7ca888fa32bf1e7d9f6950895a.php(170): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\models\FieldLayout), 'createForm', Array, 'method')
#14 /Users/chrislusk/Code/ODC/oak-street-health/vendor/twig/twig/src/Template.php(405): __TwigTemplate_6b8faab5b4449a1ff8250f619f664d06389db7f83c37ba26c78e04ead5154706->doDisplay(Array, Array)
#15 /Users/chrislusk/Code/ODC/oak-street-health/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#16 /Users/chrislusk/Code/ODC/oak-street-health/storage/runtime/compiled_templates/56/5634be2736321790da15745f87e59e70cb9397393e3c560a723e8a7c220b085c.php(124): Twig\Template->display(Array, Array)
#17 /Users/chrislusk/Code/ODC/oak-street-health/vendor/twig/twig/src/Template.php(405): __TwigTemplate_f71ef425d23b3db0d79200ba2aeaed82eb72f4c2a14fb35b375b779dcf2641bb->doDisplay(Array, Array)
#18 /Users/chrislusk/Code/ODC/oak-street-health/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#19 /Users/chrislusk/Code/ODC/oak-street-health/vendor/twig/twig/src/Template.php(390): Twig\Template->display(Array)
#20 /Users/chrislusk/Code/ODC/oak-street-health/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render(Array, Array)
#21 /Users/chrislusk/Code/ODC/oak-street-health/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render(Array)
#22 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/web/View.php(408): Twig\Environment->render('entries/_edit', Array)
#23 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/web/View.php(461): craft\web\View->renderTemplate('entries/_edit', Array)
#24 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/web/Controller.php(257): craft\web\View->renderPageTemplate('entries/_edit', Array, 'cp')
#25 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/controllers/EntriesController.php(248): craft\web\Controller->renderTemplate('entries/_edit', Array)
#26 [internal function]: craft\controllers\EntriesController->actionEditEntry(Object(craft\models\Section), 228, NULL, NULL, Object(craft\models\Site), Object(craft\elements\Entry))
#27 /Users/chrislusk/Code/ODC/oak-street-health/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#28 /Users/chrislusk/Code/ODC/oak-street-health/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#29 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction('edit-entry', Array)
#30 /Users/chrislusk/Code/ODC/oak-street-health/vendor/yiisoft/yii2/base/Module.php(534): craft\web\Controller->runAction('edit-entry', Array)
#31 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/web/Application.php(277): yii\base\Module->runAction('entries/edit-en...', Array)
#32 /Users/chrislusk/Code/ODC/oak-street-health/vendor/yiisoft/yii2/web/Application.php(104): craft\web\Application->runAction('entries/edit-en...', Array)
#33 /Users/chrislusk/Code/ODC/oak-street-health/vendor/craftcms/cms/src/web/Application.php(262): yii\web\Application->handleRequest(Object(craft\web\Request))
#34 /Users/chrislusk/Code/ODC/oak-street-health/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#35 /Users/chrislusk/Code/ODC/oak-street-health/web/index.php(22): yii\base\Application->run()
#36 {main}
```

**Comment by Kashkin** [comment]
Seeing this one too.

**Comment by khalwat** [comment]
Fixed in 3.4.10 -> https://github.com/nystudio107/craft-seomatic/releases/tag/3.4.10
